src/Application/Actions/Donations/Create.php

```diff
@@ -238,24 +250,51 @@ function () use ($donation) {
             } catch (ApiErrorException $exception) {
                 $message = $exception->getMessage();
 
-                $level = str_contains(
+                $accountLacksCapabilities = str_contains(
                     $message,
                     // this message is an issue the charity needs to fix, we can't fix it for them.
+                    // We likely want to let the team know to hide the campaign from prominents views though.
                     'Your destination account needs to have at least one of the following capabilities enabled'
-                ) ?
-                    Logger::WARNING : Logger::ERROR;
+                );
 
-                $this->logger->log($level, sprintf(
+                $failureMessage = sprintf(
                     'Stripe Payment Intent create error on %s, %s [%s]: %s. Charity: %s [%s].',
                     $donation->getUuid(),
                     $exception->getStripeCode() ?? 'unknown',
                     get_class($exception),
                     $message,
                     $donation->getCampaign()->getCharity()->getName(),
                     $donation->getCampaign()->getCharity()->getStripeAccountId() ?? 'unknown',
-                ));
-                $error = new ActionError(ActionError::SERVER_ERROR, 'Could not make Stripe Payment Intent (B)');
-                return $this->respond($response, new ActionPayload(500, null, $error));
+                );
+
+                $level = $accountLacksCapabilities ? LogLevel::WARNING : LogLevel::ERROR;
+                $this->logger->log($level, $failureMessage);
+
+                if ($accountLacksCapabilities) {
+                    /** @var string $env */
+                    $env = getenv('APP_ENV');
```

Comment:
Assertions are generally preferable to docblocks

```suggestion
                    $env = getenv('APP_ENV');
                    \assert(is_string($env));
```